README.md

```diff
@@ -0,0 +1,121 @@
+# Kubectl Wrapper Modules For Ansible
+
+The recommended approach for declarative management of Kubernetes config is
+`kubectl apply`, which contains sophisticated logic for performing a three way
+merge between an an objects incoming config, last applied config, and current
+state.  This logic however is implemented client side, and thus not available
+to callers of the API (including the current kubernetes module), and non-Go
+languages. In the future this logic may move to the API, however this is not
+expected near term.
+
+However kubectl is intended to be treated as a type of API, as such in the interim we can leverage it's logic by wrapping in Ansible modules.
+
+This prototype is specific to generically managing actual Kubernetes config, it
+does not wrap Kubernetes yaml in Ansible yaml, or expose individual modules per
+individual Kubernetes type.
+
+This prototype aims to make applying Kubernetes config via Ansible extremely
```

Comment:
Would add "consistent with the expected experience for config management in kuberentes" and link to some of brian's docs